local_store.go

```diff
@@ -431,11 +431,9 @@ func (f *fileSystemStore) Commit(consistentSnapshot bool, versions map[string]in
 			return err
 		}
 		if !info.IsDir() && isTarget(rel) && needsRemoval(rel) {
-			//lint:ignore SA9003 empty branch
 			if err := os.Remove(path); err != nil {
-				// TODO: log / handle error
+				return err
 			}
-			// TODO: remove empty directory
```

Comment:
This seems worth implementing, rather than just removing the comment?

Reply:
Addressed it 👍